tests/ci/ut_install.sh

```
@@ -4,21 +4,24 @@ set -x
-# go get github.com/docker/libtrust@latest
-# set +e
-# go get github.com/stretchr/testify@v1.8.4
 go install golang.org/x/tools/cmd/cover@latest
 go install github.com/mattn/goveralls@latest
 go install github.com/client9/misspell/cmd/misspell@latest
 set -e
 # cd ../
 # binary will be $(go env GOPATH)/bin/golangci-lint
 # go get installation aren't guaranteed to work. We recommend using binary installation.
-curl -sSfL https://raw.githubusercontent.com/golangci/golangci-lint/master/install.sh | sh -s -- -b $(go env GOPATH)/bin v2.1.2
+curl -sSfL https://raw.githubusercontent.com/golangci/golangci-lint/master/install.sh | sh -s -- -b $(go env GOPATH)/bin v2.8.0
```

Shellcheck (0.11.0)

[warning] Quote this to prevent word splitting.

(SC2046)